### ISSUE TITLE
Do not respond with a 500 error message if external APIs are down

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/sul_pub#943]
If an API request comes in to search for publications by DOI, PMID or title, we may need to make a third party API call to Web of Science or PMID to service this call.  If the 3rd party is down, we would respond with a 500 message to the caller of our API (which is the profiles system).

We should consider changing our response to a 200 with some other response (perhaps empty result set).  This requires coordination with the Profiles team to decide how to better handle this scenario.

**Comment by jcoyne** [comment]
This sounds like a perfect situation for a 503 Service Unavailable

**Comment by peetucket** [comment]
Yes, that would make more sense, though they would then have to do a better job trapping things on their end too to provide a better user experience.  But that is out of our control, and a 503 is more accurate than 500.
